aws-cognito-identity-js/build.boot

```diff
@@ -0,0 +1,47 @@
+(set-env!
+  :resource-paths #{"resources"}
+  :dependencies '[[cljsjs/boot-cljsjs "0.5.2" :scope "test"]])
```

Comment:
If you added `cljsjs/jsbn` and `cljsjs/sjcl` as dependencies here, then the users of this package wouldn't need to explicitly depend on them.